### ISSUE TITLE
How can I deduce the functions of merge layer using gradient descent?

### ISSUE THREAD

**Issue by Imorton-zd** [fchollet/keras#1377]
If I use the merge layer in keras, which paper can I refer to and how can I deduce the functions using gradient descent and represent the functions of merge layer? Thanks a lot.

**Comment by fchollet** [comment]
I don't think you need to refer to a specific paper in order to use operations such as summing tensors, or concatenating tensors. Maybe some seminal linear algebra paper?
And the gradients of these operations should be fairly obvious.
